src/common.js

```diff
@@ -0,0 +1,10 @@
+export function isValidUrl(inputValue) {
+	// Monstrosity from stackoverflow: https://stackoverflow.com/a/8234912
+	const regex = RegExp(
+		/((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=\+\$,\w]+@)?[A-Za-z0-9.-]+|(?:www.|[-;:&=\+\$,\w]+@)[A-Za-z0-9.-]+)((?:\/[\+~%\/.\w-_]*)?\??(?:[-\+=&;%@.\w_]*)#?(?:[\w]*))?)/,
+	);
+	const isValid = regex.test(inputValue);
```

Comment:
yeah just yeet these debug comments (also you have to teach me about console.debug because I never use that)